src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -63,6 +63,8 @@ def _get_feature_in_preview_message():
 
 regionsInPreview = ["ukwest", "uksouth", "westcentralus", "westus2", "canadaeast", "canadacentral", "westindia", "southindia", "centralindia"]
 
+regionsInProd = ["australiasoutheast", "northeurope", "brazilsouth", "australiaeast", "japaneast", "northcentralus", "westus", "eastasia", "eastus2", "southcentralus", "southeastasia", "eastus", "westeurope", "centralus", "japanwest"]
```

Comment:
Are there any other sources of these regions? Other than hard-coding, can we retrieve this list from other sources?

Reply:
currently, acs doesn't provide any endpoint to show available regions. This will give az cli user the better experience. So I want to keep it simple, only regions returned. I guess we can iterate on this if there is need raised.